homeassistant/components/device_tracker/google_maps.py

```diff
@@ -76,18 +79,25 @@ def _update_info(self, now=None):
                 _LOGGER.warning("No location(s) shared with this account")
                 return
 
-            attrs = {
-                ATTR_ADDRESS: person.address,
-                ATTR_FULL_NAME: person.full_name,
-                ATTR_ID: person.id,
-                ATTR_LAST_SEEN: person.datetime,
-                ATTR_NICKNAME: person.nickname,
-            }
-            self.see(
-                dev_id=dev_id,
-                gps=(person.latitude, person.longitude),
-                picture=person.picture_url,
-                source_type=SOURCE_TYPE_GPS,
-                gps_accuracy=person.accuracy,
-                attributes=attrs,
-            )
+            if self.max_gps_accuracy is not None and \
+                    person.accuracy > self.max_gps_accuracy:
+                _LOGGER.info("Ignoring %s update because expected GPS "
+                             "accuracy %s is not met: %s",
+                             person.nickname, self.max_gps_accuracy,
+                             person.accuracy)
```

Comment:
Add `return` here, then you do not need the `else`

Reply:
This is within a for loop (Looping thorough the tracked entities from Google)
Adding a return would abort the rest of the loop and entities would be missed - Have changed to a `continue` instead